src/components/Settings/Settings.tsx

```diff
@@ -1,16 +1,36 @@
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormLabel from '@mui/material/FormLabel';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
 import React from 'react';
 
 import GoToHome from 'components/Common/GoHome/GoHome';
 
 import styles from './Settings.module.scss';
 
 const Settings = () => {
+  const [value, setValue] = React.useState('LocalStorage');
```

Comment:
better import `useState` directly instead of using `React.useState()`

Reply:
ready